Airports/EGKK/Sids.txt

```diff
@@ -56,10 +59,6 @@ SID:EGKK:08L:SFD9W:D076C D344U D344M D344G SFD
 SID:EGKK:26L:TIGER1X:KKW04 KKS06 KKS09 KKS14 MAY KKS36 LAM
 SID:EGKK:26L:TIGER3M:D260C D284M D284I MAY TIGER LAM
 SID:EGKK:26R:TIGER3V:D262C D284M D284I MAY TIGER LAM
-SID:EGKK:26L:#TIGER2M:IWW23 MAY13 MAY9 MAY TIGER LAM
```

Comment:
Check whether IWW23, MAY13 and MAY9 appear in any other locations after this deletion. If they don't, I would remove them from [FIXES_General RNAV.txt](https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Navaids/FIXES_General%20RNAV.txt). If I hadn't scrolled down to the bottom, I wouldn't have noticed this... that's why you need to detail full changes made or, ideally, make issues/pull requests relate to individual changes (not individual files for the same issue obviously).